lib/cucumber/util/colors.js

```diff
@@ -1,15 +1,15 @@
-var colors = require('colors/safe');
+function Colors (useColors) {
```

Comment:
and now I must raise the question, what does this level of indirection give us :)